airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.tsx

```diff
@@ -149,33 +164,35 @@ const JobsListItem = ({ jobIndex, removeJob }: { jobIndex: number; removeJob: ()
             {({ field }: FieldProps<string>) => (
               <>
                 <label htmlFor={`jobs.${jobIndex}.job`} className={styles.jobListItemInputLabel}>
-                  Job ID
+                  <FormattedMessage id="connection.dbtCloudJobs.job.jobId" />
                 </label>
                 <Input {...field} type="text" />
               </>
             )}
           </Field>
         </div>
-        <button type="button" className={styles.jobListItemDelete} onClick={removeJob}>
+        <button type="button" className={styles.jobListItemDelete} onClick={removeJob} aria-label="Delete job">
```

Comment:
The `aria-label` should be `i18n`ed as well.

Reply:
I could be wrong - but I think we can use `<Button type="clear">` for this sort of transparent/unstyled button?